### DISCUSSION TITLE
Integration of Tensorboard

### DISCUSSION THREAD

**Discussion by Samcro5C** [timeseriesAI/tsai#633] [Q&A]
When I tried to use the fastai callback [TensorBoardCallback](https://docs.fast.ai/callback.tensorboard.html#tensorboardprojectorcallback) for a TSForecaster (minimal example from Readme), I ran into a RuntimeError (_RuntimeError: Exception occured in `TensorBoardCallback` when calling event `after_epoch`: Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead._), so I was wondering if tsai supports tensorboard at all or if this is a bug. Thank you in advance.

**Comment by oguiza** [comment]
@Samcro5C , 
I've created an issue as I've confirmed TensorBoarcdCallback is not working as expected.
